.pre-commit-config.yaml

```
@@ -0,0 +1,22 @@
+default_language_version:
+  python: python3.7
```

Since this will be running on the user's machine, it seems better to keep this as general as possible, or am I missing something? For example, specifying `python3.7` caused `pre-commit` to fail on my machine because I only have Python 3.10 installed.
```suggestion
default_language_version:
  python: python3
```

---

We need to ensure compatibility with Python 3.7. If a user submits a PR with valid Python 3.10 there is a chance that it won't be compatible with Python 3.7. For example a `match` statement

---

We need to ensure compatibility with Python 3.7. If a user submits a PR with valid Python 3.10 there is a chance that it won't be compatible with Python 3.7. For example a `match` statement